src/averageAges.js

```diff
@@ -15,11 +15,12 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
The condition in the filter function should check if the person's sex is 'm' and the century matches. The current condition is incorrect.